model/ledger/account_root.go

```diff
@@ -0,0 +1,32 @@
+package ledger
+
+import (
+	. "github.com/xyield/xrpl-go/model/transactions/types"
+)
+
+type AccountRoot struct {
+	Account           Address
+	AccountTxnID      Hash256 `json:",omitempty"`
```

Comment:
are these missing names in the json struct tags?

Reply:
not required, explained below